assets/js/script.js

```diff
@@ -32,10 +32,11 @@ var state = {
   },
   letters: 'AA',
   materials: ['white', 'white', 'white', 'white', 'white', 'white', 'white', 'white', 'white', 'white', 'white', 'white'],
```

Comment:
so ideally, materials and positions would start as an empty array. then when you add a letter, we'd do

```js
state.materials = state.materials.concat('white'); 
state.positions = state.positions.concat(0);
```

that way it's truly flexible, and not limited to an arbitrary hardcoded number of letters.